### PR TITLE
fix the bare url and the Summary of http://semver.org

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: yupengzte <yupeng36@zte.com.cn>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: VENDORING.md [hunk 1]
```diff
@@ -26,13 +26,14 @@ dependency on a package of a specific version or greater up to the next major
...
+Annotated version tags should follow [Semantic Versioning](http://semver.org) policies:
 
 "Given a version number MAJOR.MINOR.PATCH, increment the:
-    MAJOR version when you make incompatible API changes,
-    MINOR version when you add functionality in a backwards-compatible manner, and
-    PATCH version when you make backwards-compatible bug fixes.
+
+   1. MAJOR version when you make incompatible API changes,
+   2. MINOR version when you add functionality in a backwards-compatible manner, and
+   3. PATCH version when you make backwards-compatible bug fixes.
```
Comment (thaJeztah) [inline VENDORING.md]: Why change this to a numbered list?

Comment (YuPengZTE) [inline VENDORING.md]: @thaJeztah , the number list is from the http://semver.org/
sem

Comment (thaJeztah) [inline VENDORING.md]: Ah, I see. I also noticed I mis-read the diff, and thought it was a bullet-list before your change (which it wasn't) :blush:

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
thanks!

**Pull request merged**